### PR TITLE
Updating SQLite

### PR DESCRIPTION
**Pull request by abatishchev**
Cleaning up the code, using more modern C# style

### CODE REVIEW

File: docs/database-access/providers/sqlite.md [hunk 1]
```diff
@@ -132,39 +132,36 @@ C# Example (1.1 profile of the new assembly and the old assembly)
...
        IDataReader reader = dbcmd.ExecuteReader();
-       while(reader.Read()) {
-            string FirstName = reader.GetString (0);
-            string LastName = reader.GetString (1);
-            Console.WriteLine("Name: " +
-                FirstName + " " + LastName);
+       while(reader.Read())
+       {
+            string firstName = reader.GetString(0);
+            string ;astName = reader.GetString(1);
```
Comment (akoeplinger) [inline docs/database-access/providers/sqlite.md]: `;astName` should be `lastName`

Comment (abatishchev) [inline docs/database-access/providers/sqlite.md]: Oops, sorry, fixed!

**Comment by akoeplinger** [comment]
Thanks!

**Comment by abatishchev** [comment]
Awesome, thanks!

**Pull request merged**